pixi.toml

```diff
@@ -69,6 +69,24 @@ build = "*"
 [feature.build.tasks]
 build = "python -m build"
 
+[feature.benchmark]
+platforms = ["linux-64", "osx-arm64", "osx-64"]
+[feature.benchmark.dependencies]
+git_root = "*"
+python = "3.10"
+scikit-learn = ">=1.3"
+numpy = "*"
+pandas = "*"
+lightgbm = "*"
+econml = ">=0.15"
+causalml = "*"
+pip = "*"
+tabulate = "*"
+
+[feature.benchmark.tasks]
+postinstall = "pip install --no-build-isolation --no-deps --disable-pip-version-check -e ."
```

Comment:
I'm not sure about this but I think we don't need to redefine this task here, we can just use the default one (as we do in the docs environment)

Reply:
I thought so, too. Yet this hasn't worked for me:

```bash
pixi run -e benchmark postinstall
postinstall: command not found

Available tasks:
	benchmark
```

Reply:
Ah, probably because we have the flag `no-default-feature = true` for the benchmark environment